src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,15 @@
+$day-cell-size: 100px;
+$cells-gap: 1px;
+$cells-bkg-color: #eee;
+$cells-font-size: 30px;
+$field-width: ($day-cell-size + $cells-gap) * 7;
+
+$start-day: (
+  "mon": 0,
+  "tue": ($day-cell-size + $cells-gap),
+  "wed": ($day-cell-size + $cells-gap) * 2,
+  "thu": ($day-cell-size + $cells-gap) * 3,
+  "fri": ($day-cell-size + $cells-gap) * 4,
+  "sat": ($day-cell-size + $cells-gap) * 5,
+  "sun": ($day-cell-size + $cells-gap) * 6
```

Comment:
you can move calculation in the cycle where you set the correct properties for each of this modifiers